Material.Styles/Material.Styles.csproj

```diff
@@ -15,25 +15,29 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <None Remove="Fonts\**"/>
-    <AvaloniaResource Include="Fonts\**"/>
+    <None Remove="Fonts\**" />
+    <AvaloniaResource Include="Fonts\**" />
     
     <AvaloniaResource Include="**\*.xaml">
       <SubType>Designer</SubType>
     </AvaloniaResource>
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Material.Colors\Material.Colors.csproj"/>
-    <ProjectReference Include="..\Material.Ripple\Material.Ripple.csproj"/>
+    <ProjectReference Include="..\Material.Colors\Material.Colors.csproj" />
+    <ProjectReference Include="..\Material.Ripple\Material.Ripple.csproj" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Reactive"/>
+    <PackageReference Include="System.Reactive" />
   </ItemGroup>
 
   <ItemGroup>
-    <AvaloniaXaml Remove="Resources\Themes\WindowNotificationManager.axaml"/>
+    <AvaloniaXaml Remove="Resources\Themes\WindowNotificationManager.axaml" />
```

Comment:
Seems like an projects artifacts, what we don't need

Reply:
I've removed the artifacts